docs/install/README.md

```diff
@@ -0,0 +1,27 @@
+## Installing Cloud Run Events
+
+To install Cloud Run Events from master,
+
+### Using [ko](http://github.com/google/ko)
+
+```shell
+ko apply -f ./config
```

Comment:
as much as I love ko, I do think we should be releasing files (especially moving forward) that have the images resolved, etc. and should not require ko to install.

Reply:
I will add that section when I have working releases.